uktrade/data-hub-frontend
# Updated the export wins announcment banner

## Description of change

Updated the export wins announcment banner from "Important: Export Wins is moving to Data Hub on 3 April 2024" to " "Important: Export Wins is moving to Data Hub shortly".

## Test instructions

1. Go to any page in DataHub
2. You should see a banner underneath the navigation that reads "Important: Export Wins is moving to Data Hub shortly"

## Screenshots

### Before

<img width="1055" alt="Screenshot 2024-04-03 at 18 21 30" src="https://github.com/uktrade/data-hub-frontend/assets/2333157/f50b47e1-fe0d-4083-81e2-97b71ee77afa">

### After

<img width="1055" alt="Screenshot 2024-04-03 at 18 21 42" src="https://github.com/uktrade/data-hub-frontend/assets/2333157/4708fbaa-8943-40d5-ae08-cd6f62f9df85">
